lib/runner.js

```diff
@@ -118,3 +118,8 @@ Runner.prototype.run = function () {
 			stats.passCount = stats.testCount - stats.failCount;
 		});
 };
+
+// Set custom assert module
+Runner.prototype.setAssertModule = function (assertModule) {
+	Test._setAssertModule(assertModule);
```

Comment:
Why have 2 functions for a one task? You can just leave it as `setAssertModule`.